app/assets/javascripts/book_covers.js.coffee

```diff
@@ -0,0 +1,42 @@
+jQuery ->
+  window.book_cover_manager = new BookCoverManager
+class BookCoverManager
+  constructor: ->
+    this.find_book_covers()
+  find_book_covers: ->
+    number = this.isbn() || this.oclc()
+    return unless number?
+    url = "#{@google_url}#{number}"
+    $.getJSON(url, this.process_record)
+  isbn: ->
+    isbn = $("dd.blacklight-isbn_display li")
```

Comment:
I could do that - is there a concern that the ISBN in the display isn't queryable?